spi/core-spi/src/test/java/org/eclipse/dataspaceconnector/spi/types/domain/DataAddressTest.java

```diff
@@ -60,4 +60,16 @@ void verifyNullKeyThrowsException() {
                 .hasMessageContaining("Property key null.");
     }
 
+    @Test
+    void verifyGetDefaultPropertyValue() {
+        assertEquals("defaultValue", DataAddress.Builder.newInstance().type("sometype").build()
```

Comment:
We use `assertj` for assertion, I suggest to stick with it instead of using basic junit assertion methods